### ISSUE TITLE
Support custom file names for primary and secondary stem

### ISSUE THREAD

**Issue by JackismyShephard** [karaokenerds/python-audio-separator#89]
I seem to recall this project supporting custom file names for primary and secondary stem in an earlier version, but it does not seem to be supported  in the latest version (0.17.5). Would it be possible to bring that functionality back?  

I am using this project as a python library for another project, which requires a specific naming scheme for output stems, so that functionality would be greatly.

**Comment by beveradb** [comment]
Hey, sorry, I ditched that functionality once I implemented support for models which output multiple stems and stems of different types as it was messy and hard to make it consistent.

The output from `output_files = separator.separate(args.audio_file)` should give you a list of filenames though so it should be easy to rename them to whatever you want in literally one more line of python code afterwards, e.g.

```
output_files = separator.separate(args.audio_file)
output_files[0].rename('instrumental.flac')
output_files[1].rename('vocals.flac')
```

Hope that's ok!

**Comment by JackismyShephard** [comment]
@beveradb Yes, that is more or less what I am doing now. Its not a big hassle, but would be nice support custom file names.

**Comment by JackismyShephard** [comment]
@beveradb Also, great work on this project. I was looking for an interface to programmatically access the functionality of UVR and the abstractions in your API expose those both in a comprehensive and intuitive way. Also, I appreciate your documentation (although perhaps there could be more type annotations)

**Comment by beveradb** [comment]
Understood! PRs very welcome for both 😉

**Comment by hadiham** [comment]
Hello @beveradb and @JackismyShephard ! My partner @RommyT1 and I are students at the University of Michigan, and we are taking a Software Engineering class that tasked us to contribute to an open source project and we would love to contribute to issue #89!
We would appreciate it if you can assign the issue to us as we would love to implement this feature. We will make sure to review the project documentation and follow all the guidelines closely. 
Thank you for considering our request and we hope to be able to contribute soon!

**Comment by beveradb** [comment]
Think that was implemented in https://github.com/nomadkaraoke/python-audio-separator/pull/141 I'm afraid; but it's still not supported for demucs (more than 2 stem outputs) and there's basically no unit tests for anything so that would be appreciated if you're willing!

**Comment by hadiham** [comment]
Hi @beveradb 

I appreciate the opportunity to work on this issue, however the current state of the issue does not align with our requirements and goals for our class assignment. Thank you!
